docs/user/resources/01-telemetry.md

```diff
@@ -128,6 +138,16 @@ The state of the trace components is determined by the status condition of type
 | False            | ResourceBlocksDeletion               | The deletion of the module is blocked. To unblock the deletion, delete the following resources: TracePipelines (resource-1, resource-2,...) |
 | False            | MaxPipelinesExceeded                 | Maximum pipeline count exceeded                                                                                                             |
 
+Reflecting the trace data flow in the status condition is currently under development and determined by the following reasons:
+
+| Condition Status | Condition Reason  | Condition Message                                                                       |
+|------------------|-------------------|-----------------------------------------------------------------------------------------|
+| True             | FlowHealthy       | No problems detected in the trace flow                                                  |
+| False            | GatewayThrottling | Trace collector experiencing high influx: Unable to receive metrics at the current rate |
```

Comment:
```suggestion
| False            | GatewayThrottling | Trace collector experiencing high influx: Unable to receive traces at the current rate |
```

Reply:
I just copy pasted it, but you are right - we should use the term gateway everywhere, so I adapted the messages and the tests

Reply:
https://kyma-project.io/#/telemetry-manager/user/README?id=trace-gateway